privacyidea/lib/policy.py

```diff
@@ -586,8 +591,35 @@ def get_policies(self, name=None, scope=None, realm=None, active=None,
             log.debug("Policies after matching client".format(
                 reduced_policies))
 
+        if sort_by_priority:
+            reduced_policies = sorted(reduced_policies, key=itemgetter("priority"))
+
         return reduced_policies
 
+    @staticmethod
+    def check_for_conflicts(policies, action):
+        """
+        Given a (not necessarily sorted) list of policy dictionaries and an action name,
+        check that there are no action value conflicts.
+
+        This raises a PolicyError if there are multiple policies with the highest
+        priority which define different values for **action**.
+
+        Otherwise, the function just returns nothing.
+
+        :param policies: list of dictionaries
+        :param action: string
+        """
+        if policies:
+            prioritized_policy = min(policies, key=itemgetter("priority"))
+            prioritized_action = prioritized_policy["action"][action]
+            if len(policies) > 1:
```

Comment:
Can we move this up? If there is only one policy we also do not need to create prioritizes policy...

So we could do in line 613:

   if policies and len(policies) > 1: